css/webstyle.css

```diff
@@ -0,0 +1,346 @@
+body {
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  height: 7.86vh;
+  background-color: #fff;
+  align-items: baseline;
+}
+
+h3 {
+  flex: 1;
+  margin-left: 10%;
+  color: #6070ff;
+}
+
+nav {
+  flex: 2;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 10%;
+}
+
+.hide {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  padding-inline-start: 0;
+  align-self: stretch;
+}
+
+ul :first-child {
+  padding-inline-start: 0;
+}
+
+li {
+  list-style: none;
+  padding-left: 12px;
+  padding-right: 12px;
+  margin-right: 8px;
+  font-weight: 500;
+}
+
+#hamb {
+  display: none;
+}
+
+main {
+  background-color: #fff;
+  background-image: url(../images/Headerbg.png);
+  background-size: 96%;
+  background-repeat: no-repeat;
+  background-position: 0 31%;
+  border-bottom-left-radius: 8.5% 8.5%;
+}
```

Comment:
- [ ] The background image is not set correctly, please fix it.

- Your project

![your_main](https://user-images.githubusercontent.com/61562626/134771764-ed011095-49d4-402f-a923-1f4db5c2cfe2.jpeg)


- Figma template.

![main](https://user-images.githubusercontent.com/61562626/134771784-18a213e3-24ec-4ea6-b83e-6fc036bee562.jpeg)

Reply:
I can see you see in index.html file. Which I'm going to remove. After I have approved this. I put it in a git ignore so was hoping you will not see it